src/style.css

```diff
@@ -0,0 +1,142 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  box-sizing: border-box;
```

Comment:
Just set `* {box-sizing: border-box;}`. It's only rule that allows to use into `*` selector
Clean up at other selectors
```suggestion
```